subworkflows/local/vcf_normalize_bcftools/main.nf

```diff
@@ -2,9 +2,7 @@ include { BCFTOOLS_NORM                         } from '../../../modules/nf-core
 include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_1    } from '../../../modules/nf-core/bcftools/index'
 include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_2    } from '../../../modules/nf-core/bcftools/index'
 include { BCFTOOLS_INDEX as BCFTOOLS_INDEX_3    } from '../../../modules/nf-core/bcftools/index'
```

Comment:
not super fond of the _1, _2 and _3 names for the modules.
Any better ideas?

Reply:
This will be removed with the new version of all bcftools modules of nf-core with the native support of indexing in all modules.